packages/dds/tree/src/test/feature-libraries/chunked-forest/chunkEncodingEndToEnd.spec.ts

```diff
@@ -300,5 +308,83 @@ describe("End to end chunked encoding", () => {
 			// Check that the identifierValue is the original uncompressed id.
 			assert.equal(identifierValue, id);
 		});
+
+		it("In memory identifier encoding", () => {
```

Comment:
These are good test cases. However, we also want to test the case where we have a string field with a UUID (stable ID) that wasn't generated by the compressor. I'd add a third field here that has a random UUID of your choice :)